src/util/DateFormatter.js

```diff
@@ -22,8 +23,18 @@ function formatDate(longDate, dateFormat = dateConstants.LONG_DATE_FORMAT) {
  * @param format
  * @returns {string}
  */
-function getCurrentDateFormatted(format = dateConstants.LONG_DATE_FORMAT) {
+function formatCurrentDate(format = dateConstants.LONG_DATE_FORMAT) {
```

Comment:
I didn't catch this in #7, but this function is calling `formatDate(Date().valueOf(), format)`
In Firefox, it appears that `Date()` returns a local-formatted string of the current date, such as "Mon May 20 2019 14:22:35 GMT-0700 (Mountain Standard Time)", and  `Date().valueOf()` returns the same string, but `formatDate` is expecting a `long`.

Should this function return `formatDate(new Date().valueOf(), format)` instead? In Firefox, `new Date().valueOf()` returns a `long` value.

Reply:
Good catch. The `new` keyword is indeed missing.